src/features/MoviesListPage/MainContent/index.js

```diff
@@ -74,27 +73,11 @@ export const MainContent = () => {
                 content={
                     <>
                         {moviesToDisplay && moviesToDisplay.length > 0 && (
-                            moviesToDisplay.map(({
-                                title,
-                                id,
-                                vote_average,
-                                vote_count,
-                                poster_path,
-                                release_date,
-                                genre_ids
-                            }) => (
-                                <Tile
-                                    key={id}
-                                    navigateTo={() => handleTileClick(toMovie, id)}
-                                    image={`https://image.tmdb.org/t/p/w500${poster_path}`}
-                                    title={title}
-                                    subInfo={getYear(release_date)}
-                                    extraContent={
-                                        <>
-                                            <GenresList genresIds={genre_ids} />
-                                            <Rates voteAverage={vote_average} voteCount={vote_count} />
-                                        </>
-                                    }
+                            moviesToDisplay.map((movie) => (
+                                <MovieTile
+                                    key={movie.id}
+                                    movie={movie}
+                                    handleMovieClick={handleMovieClick}
```

Comment:
Hey, Tile component is generic and reusable so it's can be use as container for movie content and person content. 
Tile for movie, and for person are similar to each other so I would recommend to use  one component to wrap them. 
Tile component works like that:

-  if it received 'extraContent' prop, which contains GenresList or Rates, then it displays content for movie, 

- by default if Tile didn't received 'extraContent' prop, then it displays its content for person